cmd/gobrew/main.go

```diff
@@ -73,7 +74,14 @@ func init() {
 }
 
 func main() {
-	gb := gobrew.NewGoBrew()
+	homeDir, ok := os.LookupEnv("GOBREW_ROOT")
```

Comment:
at some point, it'd be better use a struct for all those config that use `env` into its own struct and use tags